src/internal/kopia/wrapper.go

```diff
@@ -22,6 +22,54 @@ import (
 	"github.com/alcionai/corso/src/pkg/path"
 )
 
+type IncrementalBase struct {
+	*snapshot.Manifest
+	SubtreePaths []*path.Builder
+}
+
+// PrevRefs hold the repoRef and locationRef from the items
+// that need to be merged in from prior snapshots.
+type PrevRefs struct {
+	Repo     path.Path
+	Location path.Path
+}
+
+type Backuper interface {
```

Comment:
Hah, funny enough, we're both making similar changes at the same time.  See: https://github.com/alcionai/corso/pull/2932

Thinking on the correct owner, so that these can be shared across libraries, thoughts on moving this to something like `/operations/interfaces/interfaces.go`?   Imo, it makes more sense that this is a pattern owned by operations, and that KW complies with.  In that way, we could swap out KW for another storage, or a mock, or etc.